flake.nix

```
@@ -0,0 +1,57 @@
+# This flake is supposed to enable a convenient development environment.
+# It is independent of any packaging in nixpkgs.
+#
+# See https://github.com/limine-bootloader/limine/issues/330 for more details
+# regarding the packaging in nixpkgs.
+
+{
+  description = "limeboot";
+
+  inputs = {
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    flake-parts.inputs.nixpkgs-lib.follows = "nixpkgs";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.11";
+  };
+
+  outputs = inputs@{ self, nixpkgs, flake-parts }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      flake = { };
+      # Don't artificially limit users at this point. If a build fails, they
+      # will notice it soon enough.
+      systems = nixpkgs.lib.systems.flakeExposed;
+      perSystem = { config, pkgs, ... }:
+        {
+          devShells = {
+            default = pkgs.mkShell {
+              packages = with pkgs; [
+                # Dependencies for ./bootstrap
+                autoconf
+                automake
+
+                # General build dependencies
+                cacert
+                git
```

I think we also want Make
```suggestion
                git
                gnumake
```

---

Just FYI: It's provided by default by nix, same as gcc. See below.